README.rst

```diff
@@ -23,7 +23,7 @@ cernopendata-client
 About
 =====
 
-``cernopendata-client`` is a command-like tool to facilitate downloading files
```

Comment:
@dvinayakkumar Thanks for fixing the typo. 

I have also fixed the CI issues with ReadTheDocs.